docs/_docs/install/settings.md

```diff
@@ -6,73 +6,119 @@ toc: false
+
+You will want to choose a strategy that works for your deployment, and then tweak the values to your liking before we start the application.
+For example, if you are running sregistry on a filesystem directly that you can access, you can easily write secrets into _either_:
+
+ - A `secrets.py` in the `shub/` directory alongside settings and you can use the [dummy_secrets.py](https://github.com/singularityhub/sregistry/blob/master/shub/dummy_secrets.py) as a starter template. 
+  - A `settings.yaml` in the root directory of your application (or set at `SREGISTRY_SETTINGS_FILE` and you can use the [dummy-settings.yaml](https://github.com/singularityhub/sregistry/blob/master/shub/dummy-settings.yaml) as a starter template. 
+ 
+If you are deploying via Kubernetes or similar, you can either choose to define secrets entirely in the environment, or you can have one of these files added via a config map or similar (or some combination of those two things). 
+
+## Environment
+
+As of version `2.0.0`, you can set any configuration value from settings in the environment to override the settings value.
+To determine the value for the environment variable, if it isn't already defined in the environment (e.g., `MINIO_SECRET`) 
+you can add the `SREGISTRY_` prefix to derive it. This means:
+
+ - A setting like `ENABLE_GOOGLE_AUTH` would be set as `SREGISTRY_ENABLE_GOOGLE_AUTH`
+ - To set something to True, you could do `SREGISTRY_ENABLE_GOOGLE_AUTH=True` or `SREGISTRY_ENABLE_GOOGLE_AUTH=true`
+ - To set something to False, you could do `SREGISTRY_ENABLE_GOOGLE_AUTH=False` or `SREGISTRY_ENABLE_GOOGLE_AUTH=false`
+ - To set an integer value, you could do `SREGISTRY_USER_COLLECTION_LIMIT=100` (it will be converted)
+ - To set a string value, simply set it (e.g., `SREGISTRY_REGISTRY_NAME="Tacosaurus Centers"`
+ 
+Generally, any boolean, integer, or string can be set. For database credentials, the following custom variables
+are exposed (all strings):
+
+ - SREGISTRY_DATABASE_ENGINE
+ - SREGISTRY_DATABASE_NAME (postgres)
+ - SREGISTRY_DATABASE_USER (postgres)
+ - SREGISTRY_DATABASE_HOST (db)
+ - SREGISTRY_DATABASE_PORT (5432)
+
+And you can set a custom set of _comma separated values_ (in a string) for either of `SREGISTRY_ADMINS` or `SREGISTRY_PLUGINS_ENABLED`.
+
+```console
+SREGISTRY_ADMINS="adminA,adminB,adminC"
+SREGISTRY_PLUGINS_ENABLED="ldap,google-build"
 ```
 
-Or, if you prefer a clean secrets file, create a blank one as below:
+For any setting in the sections below, you can set them in the environment (with `SREGISTRY_`)
+or via the settings.yaml file or the `secrets.py`. If you choose to use a file (and are working locally) you'll want to start by copying
+either of the templates:
 
 ```bash
-touch shub/settings/secrets.py
+cp shub/dummy_secrets.py shub/secrets.py
+cp shub/dummy-settings.yaml settings.yaml
 ```
+And make sure neither of these files are added to any kind of version control!
```

Comment:
Oh, good point. We should update `.gitignore` to now include `settings.yaml` to ignore.